external/java/src/main/java/ch/epfl/dedis/byzcoin/DataBody.java

```diff
@@ -1,6 +1,7 @@
 package ch.epfl.dedis.lib.byzcoin;
 
-import ch.epfl.dedis.proto.ByzCoinProto;
+import ch.epfl.dedis.lib.byzcoin.transaction.TxResult;
```

Comment:
this file doesn't have any changes other than an extra import, maybe we don't need it?